unlock-app/src/components/creator/CreatorLog.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import styled from 'styled-components'
+import BalanceProvider from '../helpers/BalanceProvider'
+import { dateTimeString } from '../../utils/durations'
+
+import UnlockPropTypes from '../../propTypes'
+
+/** Define default string values for the log so we don't crash the app */
+const logDefaults = {
+  DATE: 'Date Unknown',
+  NAME: '',
+  TYPE: 'TRANSACTION',
+  OWNER: '',
+  DATA: '',
+}
+
+function snakeToTitleCase(string) {
+  return string
+    .split('_')
+    .map(word => word.slice(0, 1).toUpperCase() + word.slice(1).toLowerCase())
+    .join(' ')
+}
+
+export class CreatorLog extends React.Component {
```

Comment:
Could we actually rename this `TransactionLog` (and the corresponding file). I think that would be a better description..